assets/css/app.css

```diff
@@ -18,6 +18,43 @@
   .videos-grid-with-screensharing {
     height: auto !important;
   }
+
+  .btn {
+    display: inline-block;
+    background-color: theme("colors.brand-dark-blue.500");
+    border-radius: theme("borderRadius.full");
+    color: theme("colors.brand-white");
+    line-height: 2.15rem;
+    white-space: nowrap;
+    @apply font-medium tracking-[0.05em] text-lg md:px-8 md:py-4 px-8 py-3 hover:bg-brand-white hover:text-brand-dark-blue-500 border-solid border-brand-dark-blue-500 border-2 focus:bg-brand-dark-blue-400 focus:border-brand-dark-blue-400 transition-all duration-100 ease-out;
+  }
+
+  .btn-light {
+    background-color: theme("colors.brand-white");
+    color: theme("colors.brand-dark-blue.500");
+    @apply hover:bg-brand-dark-blue-500 hover:text-brand-white border-solid border-brand-white border-2 focus:bg-brand-dark-blue-200 focus:border-brand-dark-blue-200 transition-all duration-100 ease-out;
+  }
+
+  .btn-transparent {
+    background-color: theme("colors.transparent");
+    color: theme("colors.brand-dark-blue.500");
+    @apply border-none hover:bg-transparent hover:text-brand-dark-blue-500 focus:!bg-transparent focus:!text-brand-dark-blue-500;
+  }
+
+  .btn-transparent-light {
+    background-color: theme("colors.transparent");
+    color: theme("colors.brand-white");
+    @apply border-none hover:bg-transparent hover:text-brand-white focus:!bg-transparent focus:!text-brand-white;
+  }
+
+  .btn-danger {
+    @apply bg-brand-pink-500 hover:text-brand-pink-500 focus:!text-brand-pink-500 border-solid border-brand-pink-500 border-2;
+  }
+
+  .btn-disabled {
+    background-color: theme("colors.brand-grey.60");
+    @apply pointer-events-none cursor-default border-brand-grey-60;
+  }
```

Comment:
It seems it is :wink: However, I have a different question as a Tailwind rookie - isn't that a bit against Tailwind's philosophy? There's already a JS abstraction for it - `shared/components/Button.tsx` so the argument from Tailwind docs is no longer valid

>While it’s highly recommended that you create proper template partials for more complex components, you can use Tailwind’s @apply directive to extract repeated utility patterns to custom CSS classes when a template partial feels heavy-handed.

The props here could be handled as properties of `<Button>` instead of requiring to know the CSS className for it. Here's a lib that's using such an approach: https://www.material-tailwind.com/docs/react/button

Reply:
That's a very good point. It's already improved